docs/getting_started/faq.rst

```diff
@@ -89,15 +89,16 @@ import with Modin import:
 
 Which execution engine (Ray or Dask) should I use for Modin?
 """"""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-
-Whichever one you want! Modin supports Ray_ and Dask_ execution engines to provide an effortless way 
-to speed up your pandas workflows. The best thing is that you don't need to know 
-anything about Ray and Dask in order to use Modin and Modin will automatically 
-detect which engine you have 
-installed and use that for scheduling computation. If you don't have a preference, we recommend 
-starting with Modin's default Ray engine. If you want to use a specific 
-compute engine, you can set the environment variable ``MODIN_ENGINE`` and 
-Modin will do computation with that engine:
+If you are on Windows, you must use Dask as Ray does not support Windows. If you are on
+Linux or Mac OS, you can install and use either engine. 
+
+Modin supports Ray_ and Dask_ execution engines to provide an effortless way 
+to speed up your pandas workflows. The best thing is that there is no pre-requisite 
+knowledge required to use either of these engines as Modin abstracts away all of the 
+complexity. Modin will automatically detect which engine you have installed and use that for scheduling computation. 
```

Comment:
```suggestion
to speed up your pandas workflows. You don't need to know anything about either engine in order to use it with Modin. If you only have one engine installed, Modin will automatically detect which engine you have installed and use that for scheduling computation.
```

I prefer something closer to the original phrasing, like this. Why change it? I vote for deleting "the best thing" because this is not necessarily the best thing and whether it's the best thing is not relevant for the point we should be making here, which is that you don't have to know the details of either engine to use it with Modin.